src/app/main.rs

```diff
@@ -379,7 +384,10 @@ where
                     ));
 
                     rt.spawn(tap_daemon.map_err(|_| ()));
```

Comment:
Because we are disabling tap for the entire proxy, we could have a similar `if let` check here and conditionally spawn the `tap_daemon`.